lesson2.js

```diff
@@ -0,0 +1,58 @@
+const welcomeUser = () => {
+  const name = prompt("Enter your name!");
+  const surname = prompt("Enter your surname!");
+  const upperName = name[0].toUpperCase() + name.slice(1);
+  const upperSurname = surname[0].toUpperCase() + surname.slice(1);
+
+  alert(`Hello ${upperName} ${upperSurname}`);
+}
+
+const compareStrings = () => {
+  const password = prompt("Enter your password!");
+  const confirmPassword = prompt("Confirm your password!");
+
+  alert(password === confirmPassword ? 'Password validated' : 'Password do not match');
+}
```

Comment:
const compareStringsAndReturnMessage = (password, confirmPassword) => {
  return password === confirmPassword ? 'Password validated' : 'Password do not match');
}

  const password = prompt("Enter your password!");
  const confirmPassword = prompt("Confirm your password!");
  const message = compareStringsAndReturnMessage(password, confirmPassword)